.circleci/config.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       - run:
           name: Check frontend test coverage
           command: |
-            python -m scripts.check_frontend_coverage
+            python -m scripts.run_frontend_tests --run_minified_tests --skip_install --check_coverage
```

Comment:
I think this flag should just be added to the frontend_tests stanza, and the frontend_test_coverage stanza can be fully deleted.

Reply:
Sorry I didn't get it.

Reply:
Add the flag to line 100 and remove this block (lines 108-119).

Reply:
@kevinlee12 Thanks! Done.